src/dos/drive_overlay.cpp

```diff
@@ -1230,6 +1260,10 @@ bool Overlay_Drive::Rename(const char * oldname, const char * newname) {
 
 		result = !ec; // success if no error-code
 
+		if (result == true) {
```

Comment:
`if (result)` or better yet, also refactor `result` to `success`.

Reply:
fixed